### DISCUSSION TITLE
请问profiles.conf文件里面预设组自动加载滤镜mvtools_2x.vpy怎么写

### DISCUSSION THREAD

**Discussion by github-xihe** [hooke007/mpv_PlayKit#262] [mpv 通用]
[VCB-Studio]
 profile-desc = VCB-Studio
 profile-cond = filename:match("VCB%-Studio")~=nil
 profile-restore = copy
 glsl-shaders = "~~/shaders/KrigBilateral.glsl"
 glsl-shaders-append = "~~/shaders/AMD_FSR_EASU_rgb.glsl"
 glsl-shaders-append = "~~/shaders/Anime4K_Restore_CNN_L.glsl"
 
 我想播放选定字幕组的视频时着色器和补帧滤镜随播放一起自动开启。

**Comment by hooke007** [comment]
有没有可能，仓库里的 mpv.conf 就有示例

**Comment by github-xihe** [comment]
弄了挺久弄出来了  vf = vapoursynth="~~/vs/mvtools_2x.vpy"   第二个=不能有空格

**Reply by hooke007** [reply]
空格不是随意给的。
`vapoursynth=xxxxxxxx:xxxx=xxx:xxx=xxx` 本身就是此滤镜的完整写法
https://github.com/hooke007/MPV_lazy/discussions/120 这里有足够多的案例体现
